cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/InstanceStatus.java

```diff
@@ -6,6 +6,7 @@ public enum InstanceStatus {
     STARTED(StatusGroup.PERMANENT),
     STOPPED(StatusGroup.PERMANENT),
     FAILED(StatusGroup.PERMANENT),
+    PENDING(StatusGroup.PERMANENT),
```

Comment:
Just had one q, should this be `PERMANENT` or `TRANSIENT`

Reply:
Changed to TRANSIENT as PENDING won't be a final state.